packages/ui-shared/package.json

```diff
@@ -50,11 +50,11 @@
     "dist"
   ],
   "dependencies": {
+    "@botpress/studio-be": "*",
```

Comment:
Why does ui-shared depend on the studio's backend?

Reply:
Because it uses common files which are generated in the backend. Eventually i'd like to move those common files in a "common" package (once the typings submodule is added, because theres a couple of sdk typings from the sdk)

That dependency was always there, but we controlled it with the gulp parallel/series, it's much better to have a straightforward dependency